README.md

```diff
@@ -1,52 +1,147 @@
-## BEGIN - TO DELETE TILL END
+# reshuffle-jira-connector
 
-THIS IS A TEMPLATE REPO FOR NEW RESHUFFLE CONNECTORS
-1. Create a new connector repo from this template using this link https://github.com/reshufflehq/reshuffle-template-connector/generate
-2. Clone the repo locally
-3. Rename all occurrences of _CONNECTOR_NAME_
-4. `npm install`
-5. `npm run build:watch`
-6. Implement your events/actions in `src/index.ts`
-7. `npm run lint`
-8. Push your code
-9. Go to https://app.circleci.com/projects/project-dashboard/github/reshufflehq/
-    a. You should see your new connector repo
-    b. click on `Set Up Project` for the repo
-    c. click on `Use Existing Config`
-    d. click on `Start Building`
+[Code](https://github.com/reshufflehq/reshuffle-jira-connector) |  [npm](https://www.npmjs.com/package/reshuffle-jira-connector) | [Code sample](https://github.com/reshufflehq/reshuffle/tree/master/examples/jira)
 
-10. If circle CI checks are all green, you are all set!
+`npm install reshuffle-jira-connector`
 
-// Keep documentation template below
+This connector uses [Node Jira Client](https://github.com/jira-node/node-jira-client) package.
 
-## END
+### Reshuffle Jira Connector
 
-# reshuffle-_CONNECTOR_NAME_-connector
+This connector provides a connector for [Atlassian Jira](https://www.atlassian.com/software/jira).
 
-### Reshuffle _CONNECTOR_NAME_ Connector
+Create an API token from your Atlassian account:
 
-This connector provides <description>.
+1. Log in to https://id.atlassian.com/manage/api-tokens.
+2. Click Create API token.
+3. From the dialog that appears, enter a memorable and concise Label for your token and click Create.
+4. Click Copy to clipboard, then paste the token to your script, or elsewhere to save
 
 #### Configuration Options:
+
+There are several ways to instantiate this connector (see options below).
```

Comment:
What do you mean by several ways? If it's "here are the required fields but there are others optional" we might want to say something to that effect. Otherwise, I'm looking for maybe alternative ways to authenticate

Reply:
good point, I've changed it